dev-ml/ocaml-fileutils/ocaml-fileutils-0.6.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DUNE_PKG_NAME="fileutils"
+
+inherit dune
+
+# Currently tests trigger some compiler bug.
```

Comment:
I'd mention some details what kind of bug is that, preferably with a link to an upstream bug tracker

Reply:
Removed, problem may have been on my end, tests run fine now in any case.